### PR TITLE
Sort and copyedit changelog.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,29 +1,27 @@
...
 Important Bugfixes:
  * Add missing callvalue check to the creation code of a contract that does not define a constructor but has a base that does define a constructor.
- * Disallow index range accesses for arrays with dynamically encoded base types.
- * Code Generator: Fixed that string literals containing backslash characters could cause incorrect code to be generated when passed directly to function calls or encoding functions when ABIEncoderV2 is active.
+ * Disallow index range (array slice) accesses for arrays with dynamically encoded base types.
+ * Fixed that string literals containing backslash characters could cause incorrect code to be generated when passed directly to function calls or encoding functions when ABIEncoderV2 is active.
 
 
 Language Features:
  * Implemented ``type(X).min`` and ``type(X).max`` for every integer type ``X`` that returns the smallest and largest value representable by the type.
```
Comment (axic) [inline Changelog.md]:
```suggestion
 * Implemented ``type(T).min`` and ``type(T).max`` for every integer type ``T`` that returns the smallest and largest value representable by the type.
```

File: Changelog.md [hunk 2]
```diff
@@ -1,29 +1,27 @@
-### 0.6.8 (unreleased)
+### 0.6.8 (2020-05-14)
 
 Important Bugfixes:
  * Add missing callvalue check to the creation code of a contract that does not define a constructor but has a base that does define a constructor.
- * Disallow index range accesses for arrays with dynamically encoded base types.
- * Code Generator: Fixed that string literals containing backslash characters could cause incorrect code to be generated when passed directly to function calls or encoding functions when ABIEncoderV2 is active.
+ * Disallow index range (array slice) accesses for arrays with dynamically encoded base types.
+ * Fixed that string literals containing backslash characters could cause incorrect code to be generated when passed directly to function calls or encoding functions when ABIEncoderV2 is active.
```
Comment (ekpyron) [inline Changelog.md]:
```suggestion
 * Disallow array slices stemming from index range accesses to arrays with dynamically encoded base types.
 * String literals containing backslash characters can no longer cause incorrect code to be generated when passed directly to function calls or encoding functions when ABIEncoderV2 is active.
```
Maybe, if we're changing them anyways :-)?

Or for the first one:
``* Disallow array slices of arrays with dynamically encoded base types.``

File: Changelog.md [hunk 3]
```diff
@@ -1,29 +1,27 @@
...
 
 
 Language Features:
  * Implemented ``type(X).min`` and ``type(X).max`` for every integer type ``X`` that returns the smallest and largest value representable by the type.
 
 
 Compiler Features:
- * Commandline Interface: Don't ignore `--yul-optimizations` in assembly mode.
```
Comment (axic) [inline Changelog.md]: Why remove this?

**Review by ekpyron — DISMISSED** [review]

File: Changelog.md (hunk 3 above)
Comment (ekpyron) [reply Changelog.md]: Wow, I just overlooked that!

Comment (chriseth) [reply Changelog.md]: merge problem I think

**Review by ekpyron — APPROVED** [review]

**Pull request merged**